README.md

```diff
@@ -26,16 +26,21 @@ Fetches and creates versioned GitHub resources.
 * `insecure`: *Optional. Default `false`.* When set to `true`, concourse will allow
   insecure connection to your github API.
 
-* `release`: *Optional. Default `true`.* When set to `true`, `put` produces
-  release and `check` detects releases.  If `false`, `put` and `check` will ignore releases.
-  Note that releases must have semver compliant tags to be detected.
-
-* `pre_release`: *Optional. Default `false`.* When set to `true`, `put` produces
-  pre-release and `check` detects prereleases. If `false`, only non-prerelease releases
-  will be detected and published. Note that releases must have semver compliant
-  tags to be detected.
-  If `release` and `pre_release` are set to `true`, `put` produces
-  release and `check` detects prereleases and releases.
+* `release`: *Optional. Default `true`.* When set to `true`, `check` detects
+  final releases and `put` publishes final releases (as opposed to
+  pre-releases). If `false`, `check` will ignore final releases, and `put` will
+  publish pre-releases if `pre_release` is set to `true`
+  **note:** if both `release` and `pre_release` are set to `false`, `put` will
+  still produce final releases
+  **note:** releases must have semver compliant tags to be detected.
+
+* `pre_release`: *Optional. Default `false`.* When set to `true`, `check`
+  detects pre-releases, and `put` will produce pre-releases (if `release` is
+  also set to `false`). If `false`, only non-prerelease releases will be detected
+  and published.
+  **note:** if both `release` and `pre_release` are set to `true`, `put`
+  produces final releases and `check` detects both pre-releases and releases.
```

Comment:
I didn't notice in the beginning that this is a different note, do you think it would be better to have both notes in the same place?

Reply:
Ah good point catch - I was hoping they'd appear on separate lines

I put line breaks between the notes and the body - let me know what you think

Reply:
when I read my comment again I noticed it is very vague 😁.
what I meant is to have all the notes related different setups in the same place, I will add it as a suggestion.